CMakeLists.txt

```diff
@@ -1,20 +1,19 @@
 CMAKE_MINIMUM_REQUIRED(VERSION 2.8.5)
+
+option(SAS "Whether to run static code analysis with SAS." OFF)
+if (SAS)
+    find_package(sas)
+    enable_sas(
+        FORMAT
+        COMPARISON_REPORT_DIR "${CMAKE_BINARY_DIR}/sas_report"
+        IGNORE_DIRS ${CMAKE_SOURCE_DIR}/build ${CMAKE_SOURCE_DIR}/cmake ${CMAKE_SOURCE_DIR}/doc
```

Comment:
However, I'd like to keep this in, because it also allows to ignore all other build directories (i.e. if you have both debug and release builds).